e4-osc-bridge.py

```diff
@@ -95,17 +105,18 @@ def start_replay(replay_file, osc_ip, osc_port):
     osc_client = SimpleUDPClient(osc_ip, osc_port)
 
     log_file = open(args.log_file, "r")
-    last_time = 0
+    last_time = {}
     for line in log_file:
         line = line.strip()
         event_time, event_type, *sample = line.split(",")
         event_time = float(event_time)
         sample = [float(x) for x in sample]
 
-        time.sleep(event_time - last_time)
-        last_time = event_time
+        # print(event_time, last_time)
+        time.sleep(abs(event_time - last_time.get(event_type,0)))
+        last_time[event_type] = event_time
```

Comment:
I'm not so sure about this part... All the events should ideally be sorted in the file per date, so sleeping should happen until the next event, not the next event of that type.